spark.md

```diff
@@ -62,21 +41,23 @@ The first step is to copy a BAM file to HDFS, which can be achieved using the `h
 hadoop fs -put ./src/test/resources/org/broadinstitute/hellbender/tools/flag_stat.bam flag_stat.bam
 ```
 
-Next, run `spark-submit` again, but with a few extra configuration properties set, which are suitable for a cluster
-environment:
+Now, when you run your GATK Spark tool you add Spark-related configuration options after the tool-specific
+options. Note that the tool options and the Spark options are separated by an extra `--`:
 
 ```bash
 NAMENODE=<hostname of the HDFS namenode>
 SPARK_MASTER=yarn-client
-spark-submit \
-  --master $SPARK_MASTER \
-  --conf spark.driver.userClassPathFirst=true \
-  --conf spark.executor.userClassPathFirst=true \
-  --conf spark.io.compression.codec=lzf \
-  build/libs/gatk-all-*-spark.jar CountReadsSpark \
+
+gatk-launch \
+    CountReadsSpark \
     --input hdfs://$NAMENODE/user/$USER/flag_stat.bam \
     --output hdfs://$NAMENODE/user/$USER/out/countreads \
-    --sparkMaster $SPARK_MASTER
+    -- \
+    --sparkRunner SUBMIT --sparkMaster $SPARK_MASTER \
+    --conf spark.yarn.executor.memoryOverhead=600 \
```

Comment:
After discussion with @akiezun I am going to just delete this spark.md file and move the 1 or 2 useful tidbits to the README, since as it currently stands 90% of it just rehashes what the README says